modules/ROOT/pages/manage-policies-overview.adoc

```diff
@@ -0,0 +1,54 @@
+= Gateway Policies in API Mananager
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:page-aliases: 
+
+From API Mananager, you can apply policies to APIs in Flex Gateway Connected Mode or in Mule Gateway.
+
+Policies enable you to enforce regulations to help manage security, control traffic, and improve adaptability of your APIs. For example, a policy can control authentication, access, allotted consumption, and service level access (SLA).
+
+With policies, you can implement these regulations with no modification to the code implementation. 
+
+Because the policies are implemented by the Gateway, policies are documented in the xref:gateway-home::gateway-overview.adoc[Gateway documentation]. For more information, see the xref:policies::policies-overview.adoc[Policy documentation] or use the links in the following sections.
+
+== Policies Included in Flex Gateway and Mule Gateway
+
+* xref:policies::policies-included-directory.adoc[Included Policies Directory]
+
+== Applying Policies in Flex Gateway Connected Mode and in Mule Gateway
+
+* xref:policies::policies-included-apply.adoc[Apply an Included Policy]
+
+== See Also
+
+* xref:policies::policies-included-reorder.adoc[]
```

Comment:
just checking if you wanted to include a title in these brackets

Reply:
Yeah, it just renders as the page title